.doc_gen/metadata/ec2_metadata.yaml

```diff
@@ -1235,7 +1235,7 @@ ec2_DescribeInstances:
           excerpts:
             - description:
               snippet_tags:
-                - ec2.java2.scenario.describe_instance.main
+                - ec2.java2.running_instances.main

```

Comment:
why is the name of this snippet being changed?

Reply:
That was an intentional change. It works too as I triple checked the CD build.  Tag describes the code in more detail.

Reply:
But it's now completely inconsistent with _every other snippet in for that action_. If it's a better description, then do we need to change the snippets for other languages? Why is "running_instances" a better description when the action being tagged is called "DescribeInstances?"